tensorflow/lite/tools/cmake/modules/xnnpack/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2022?

Reply:
Yes, this PR was filed on December 16th 2021 - see first comment in this thread (https://github.com/tensorflow/tensorflow/pull/53449#issue-1081883459) . 
Anyway, updated to 2022 to match the eventual merge date.